flexget/plugins/sites/torrentday.py

```diff
@@ -0,0 +1,184 @@
+              tv480p, tvBRD, tvDVD, tvDVDrip,
+              tvMOBILE, tvPACKS, tvSDx264, 
+              tvHDx264, tvX265, tvXVID
+    """
+
+    schema = {
+        'type': 'object',
+        'properties': {
+            'rss_key': {'type': 'string'},
+            'uid': {'type': 'string'},
+            'passkey': {'type': 'string'},
+            'cfduid': {'type': 'string'},
+            'category': {
+                'oneOf': [
+                    {'type': 'integer'},
+                    {'type': 'string', 'enum': list(CATEGORIES)},
+                ]
+            },
+        },
+        'required': ['rss_key', 'uid', 'passkey', 'cfduid'],
+        'additionalProperties': False
+    }
+
+    # urlrewriter API
+    def url_rewritable(self, task, entry):
+        url = entry['url']
+        if url.find('.torrent'):
+            return False
+        if url.startswith('https://www.torrentday.com'):
+            return True
+        return False
+
+    # urlrewriter API
+    def url_rewrite(self, task, entry):
+        if 'url' not in entry:
+            log.error("Didn't actually get a URL...")
+        else:
+            log.debug("Got the URL: %s", entry['url'])
+        if entry['url'].startswith('https://www.torrentday.com/browse'):
+            # use search
+            results = self.search(task, entry)
+            if not results:
+                raise UrlRewritingError("No search results found")
+            entry['url'] = results[0]['url']
+
+    @plugin.internet(log)
+    def search(self, task, entry, config=None):
+        """
+        Search for name from torrentday.
+        """
+
+        if not isinstance(config, dict):
+            config = {}
+        categories = config.get('category', 'all')
+        # Make sure categories is a list
+        if not isinstance(categories, list):
+            categories = [categories]
+        # If there are any text categories, turn them into their id number
+        categories = [c if isinstance(c, int) else CATEGORIES[c] for c in categories]
+        filter_url = '&cata=yes&c%s=1&clear-new=1' % ','.join(str(c) for c in categories)
+        entries = set()
+        for search_string in entry.get('search_strings', [entry['title']]):
+            query = normalize_unicode(search_string).replace(":", "")
+            # urllib.quote will crash if the unicode string has non ascii characters, so encode in utf-8 beforehand
+            url = ('https://www.torrentday.com/browse.php?search=' +
+                   quote(query.encode('utf-8')) + filter_url)
+            log.debug('Using %s as torrentday search url', url)
+
+            cookies = {}
+            cookies["uid"] = config['uid']
+            cookies["pass"] = config['passkey']
+            cookies["__cfduid"] = config['cfduid']
+
+            try:
+                page = requests.get(url, cookies=cookies).content
+            except RequestException as e:
+                raise PluginError('Could not connect to torrentday')
```

Comment:
You could've changed it to `raise PluginError('Could not connect to torrentday: %s' % str(e))`